feature/tabsflow/tabs/src/main/java/com/epmedu/animeal/tabs/TabsHost.kt

```diff
@@ -86,5 +90,24 @@ private fun NavigationTabs(navigationController: NavHostController) {
         screen(NavigationTab.Home.route.name) { HomeScreen() }
         screen(NavigationTab.Analytics.route.name) { AnalyticsScreen() }
         screen(NavigationTab.More.route.name) { MoreHost() }
+        screen(
+            "${WillFeedRoute.Confirmation.name}/{comesFromHome}",
+            arguments = listOf(navArgument("comesFromHome") { type = NavType.BoolType })
+        ) { backStackEntry ->
+            backStackEntry.arguments?.getBoolean("comesFromHome")?.let { comesFromHomeScreen ->
+                FeedConfirmationScreen(
+                    {
+                        if (comesFromHomeScreen) {
+                            navigationController.popBackStack()
+                        } else {
+                            navigationController.navigate(NavigationTab.Home.route.name)
+                        }
+                    },
+                    {
+                        navigationController.popBackStack()
+                    }
+                )
+            }
+        }
```

Comment:
I don't see why did you implement all this logic, if cancel button should just use popBackStack and agree button always leads to the same screen

Reply:
You are right. I changed the logic and the parameter from Boolean to Int representing the id of the feeding point selected. This should be useful when the user presses **I will feed button** from search and favourites tab so the HomeScreen knows that it has to change their state to **_showing path to feeding point_** or something like that.

Reply:
We strive to avoid passing parameters in our screen composables. Incapsulate this logic inside the screen implementaion and if we need some branching we will think about some better solution

Reply:
What do you think about this:

```
@Composable
private fun NavigationTabs(navigationController: NavHostController) {
    ScreenNavHost(
        navController = navigationController,
        startDestination = NavigationTab.Home.route.name
    ) {
        screen(NavigationTab.Search.route.name) { SearchScreen() }
        screen(NavigationTab.Favorites.route.name) { FavouritesScreen() }
        screen(NavigationTab.Home.route.name) { HomeScreen() }
        screen(NavigationTab.Analytics.route.name) { AnalyticsScreen() }
        screen(NavigationTab.More.route.name) { MoreHost() }
        screen(TabsRoute.FeedConfirmation.name) {
            FeedConfirmationScreen(
                onAgreeClick = { navigationController.navigate(NavigationTab.Home.route.name) },
                onCancelClick = { navigationController.popBackStack() }
            )
        }
    }
}
```